scripts/fix_wheel.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+import zipfile
+from glob import glob
+
+if __name__ == "__main__":
+    # Due to https://github.com/python-poetry/poetry/issues/3509 and Poetry
+    # assuming the tags, we have to change the wheel ourselves after build
```

Comment:
Well this is too bad.

Can we just build the wheels directly rather than via poetry? Will that still have this issue, or is it too inconvenient to do in the first place?

Reply:
I kinda am, but you need Poetry to get all the files that you want in the wheel setup properly and to have the dependency versions set in the dist info properly.